tests/base.py

```diff
@@ -8,6 +8,15 @@
 from datetime import timedelta
 
 from tap_tester.base_suite_tests.base_case import BaseCase
+from tap_tester.jira_client import JiraClient as jira_client
+from tap_tester.jira_client import CONFIGURATION_ENVIRONMENT as jira_config
+
+JIRA_CLIENT = jira_client({ **jira_config })
+
+def get_jira_card_status(jira_ticket):
+    """Get the status of a specified JIRA card"""
+    jira_status = JIRA_CLIENT.get_jira_issue_status(jira_ticket)
+    return jira_status
```

Comment:
This function is too simple to require a docstring. And this docstring doesn't add anything that we don't get from reading the name of the function or reading how the internals of this function.

It's also just a passthrough function. If this was written just to shorten the name whenever the function needs to used, then we can just define it like this to get the exact same effect
```suggestion
get_jira_card_status = JIRA_CLIENT.get_jira_issue_status
```